packages/console/src/pages/Dashboard/components/Error/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import ErrorImage from '@/assets/images/table-error.svg';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  errorMessage?: string;
+};
+
+const Error = ({ errorMessage }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  return (
+    <div className={styles.error}>
+      <img src={ErrorImage} />
+      <div className={styles.title}>{errorMessage ?? t('errors.something_went_wrong')}</div>
+    </div>
+  );
+};
```

Comment:
Can we just re-use `<AppError>` component?

Reply:
updated